src/cgame/cgame.h

```diff
@@ -595,7 +595,7 @@ typedef struct cg_import_s {
 	 * @return The linked entity.
 	 * @remarks The parent entity must have been previously returned by `AddEntity`.
 	 */
-	const r_entity_t *(*AddLinkedEntity)(const r_entity_t *parent, const r_model_t *model, const char *tag_name);
+	r_entity_t *(*AddLinkedEntity)(const r_entity_t *parent, const r_model_t *model, const char *tag_name);
```

Comment:
Basically, yes. I can see if I can work around this if you like, but the issue was mainly that AddLinkedEntity didn't give us a chance to set up the matrix properly. If I moved AddLinkedEntity to the cgame it might be possible to return these to const.